app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -220,8 +229,16 @@ def handle_failed_status(enrollment, response)
     enrollment.update(status: :failed)
     if response['fraudSuspected']
       send_failed_fraud_email(enrollment.user, enrollment)
+      analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_email_initiated(
+        **email_analytics_attributes(enrollment),
+        email_type: 'Failed fraud suspected email type',
```

Comment:
The trailing " email type" in these strings feels unnecessary since it's already part of the property name.

```suggestion
        email_type: 'Failed fraud suspected',
```